crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -163,7 +163,7 @@ impl Plugin for MeshRenderPlugin {
                 .add_systems(
                     Render,
                     (
-                        set_mesh_motion_vector_flags.before(RenderSet::Queue),
+                        set_mesh_motion_vector_flags.in_set(RenderSet::PrepareAssets),
```

Comment:
I don’t think the systems have to be in the same set anymore to have edges between them? Or is there something else requiring moving between sets?

Reply:
I just thought it was cleaner to put them in the same set.